examples/nlp/gpt/conf/gpt_ppo_critic.yaml

```diff
@@ -60,6 +64,10 @@ model:
     mean: null
     std: null 
 
+  # if the inference microbatch size is big, it's possible
+  # to split it using forward mbs and run inference iteratively
+  forward_micro_batch_size: ${.micro_batch_size}
```

Comment:
I added the comment for this in the inference micro batch size. The reason is because of pipeline parallelism. When pipeline parallelism is enabled we want PyTriton to give us as big of a batch as we can(but not too big where it just waits for too long). then we can cut this up to smaller micro batches and then have good GPU utilization that way(since we need to fill the pipe each time we do a rollout).

so in a way inference micro batch size is telling pytriton what batch sizes we like, and that can be a list. once we tell them this we will then cut this batch to smaller batches

Reply:
Got it, thanks!